lib/logstash/inputs/beats.rb

```diff
@@ -189,6 +195,14 @@ def register
       @logger.warn("configured ssl_key => #{@ssl_key.inspect} will not be used") if @ssl_key
     end
 
+    resolve_enriches
```

Comment:
In case of naming convention `resolved_enriches`, it is really up to team/developers but I usually use the _past_ tense if I declare static values and make naming _actionable_ ex.: `doSomething/do_something` if method executes some actions. And, in our source base I do see similar approach, ex.: `create_server`. It is minor but sometimes brings some confusions. Let me please know your opinion.